docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Marketplace Product Approval Process feature overview
+description: This document contains concept information for the Marketplace Product Approval Process feature.
+template: concept-topic-template
+---
+The [marketplace operator](/docs/marketplace/user/intro-to-the-spryker-marketplace/back-office-for-marketplace-operator.html) is primarily responsible for ensuring the quality of data on the marketplace, including merchants, products, and offers. To control those things in the Marketplace, the approval mechanism is a key feature.
+
+The Marketplace Product Approval Process allows marketplace owners to verify the products of merchants before showing them to customers.
+
+Thus, only the approved and active products within the validity dates range and existing for the defined store are displayed in the Storefront.
```

Comment:
Thus, only the approved, active, within the validity dates range, and existing for the defined store are displayed in the Storefront.